### PR TITLE
comment install instructions for archlinux until updated

### PR DESCRIPTION
**Pull request by to266**
for https://github.com/hyperspy/hyperspy/issues/1069

### CODE REVIEW

**Comment by francisco-dlp** [comment]
@jat255, could you have a look at this?

**Comment by jat255** [comment]
Looks good to me. For the time being, hopefully most Arch users would know how to install using `pip`, but I will revise these instructions after my defense (mid-July).

**Comment by to266** [comment]
FYI I just commented-out the current instructions so that they don't confuse people - any other change is purely due to alignment and similar things

File: doc/user_guide/install.rst [hunk 1]
```diff
@@ -223,25 +224,29 @@ In Debian/Ubuntu you can install the libraries as follows:
...
+    In Arch Linux, the following command should install the required packages
+    to get a fully functional installation:
+    
+    .. code-block:: bash
+    
+        $ sudo pacman -Sy python2 python2-numpy	python2-matplotlib	python2-pip
+        python2-traits python2-traitsui python2-h5py python2-scikit-learn
+        python2-nose python2-statsmodels python2-pillow python2-pyqt4
+        python2-pyqt5 python2-scipy python2-pandas python2-setuptools ipython2
+        python2-jinja python2-pyzmq python2-pyqt4 python2-tornado python2-sip
```
Comment (dnjohnstone) [inline doc/user_guide/install.rst]: Should this all still be python2? Surely python3 - or did I miss something basic?

Comment (to266) [inline doc/user_guide/install.rst]: They are (or should be!) commented out, i.e. not visible. The instructions being for python2 is precisely the reason of this PR - to hide them and not confuse people.

**Pull request merged**